pkg/controller/nodemaintenance/check_lease_update_test.go

```diff
@@ -404,7 +404,7 @@ var _ = Describe("checkLeaseUpdate", func() {
 						HolderIdentity:       makeString(LeaseHolderIdentity),
 						LeaseDurationSeconds: makeInt32(LeaseDurationInSeconds),
 						AcquireTime:          nil,
-						RenewTime:            getTimeLease(int64(NowTime) - LeaseDurationInSeconds + 2 * drainerTimeoutInSeconds - 1 ),
+						RenewTime:            getTimeLease(int64(NowTime) - LeaseDurationInSeconds + 2 * int64(drainerTimeout) / int64(time.Second) - 1 ),
```

Comment:
`getTimeLease()` takes milliseconds as argument, right?
I'm a bit confused about using `LeaseDurationInSeconds` then.
And for getting the milliseconds of a Duration you can use `drainerTimeout.Milliseconds()` instead of 
`int64(drainerTimeout) / int64(time.Second)` (Second is confusing here as well)